cmd/trade.go

```diff
@@ -496,10 +501,44 @@ func convertDeprecatedBotConfigValues(l logger.Logger, botConfig trader.BotConfi
 
 func runTradeCmd(options inputs) {
 	l := logger.MakeBasicLogger()
-	botConfig := readBotConfig(l, options)
+	botStart := time.Now()
+	botConfig := readBotConfig(l, options, botStart)
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
+	// TODO DS Fail if in release mode with undefined API key.
+
+	userID := "-1" // TODO DS Properly generate and save user ID.
+	httpClient := &http.Client{}
+	var guiVersionFlag string
+	if *options.ui {
+		guiVersionFlag = guiVersion
+	}
+
+	metricsTracker, e := metrics.MakeMetricsTracker(
+		userID,
+		amplitudeAPIKey,
+		httpClient,
+		botStart,
+		version,
+		runtime.GOOS,
+		runtime.GOARCH,
+		"", // TODO DS Determine how to get GOARM.
```

Comment:
`""` is a valid value for GOARM in the future (when it's missing).
so since it's unknown at this point, can you call it "unknown" so we don't incorrectly assign it as the valid value `""`

Reply:
alternatively we can choose a different value to represent a non-arm system such as `none`, your call but something to keep in mind for when this is addressed in the future

Reply:
Good call! Called it `unknown_goarm` just in case `unknown` is also valid - this should clearly disambiguate it.